### PR TITLE
Don't ask for pass after completing setup

### PR DESCRIPTION
**Pull request by alexfmpe**

### CODE REVIEW

File: desktop/src/Desktop/Setup.hs [hunk 1]
```diff
@@ -128,7 +128,7 @@ kadenaWalletLogo = divClass "logo" $ do
   elClass "div" "chainweaver" $ text "Chainweaver"
   elClass "div" "by-kadena" $ text "by Kadena"
 
-type SetupWF t m = Workflow t m (WalletScreen, Event t Crypto.XPrv)
+type SetupWF t m = Workflow t m (WalletScreen, Event t (Crypto.XPrv, Text))
```
Comment (mankyKitty) [inline desktop/src/Desktop/Setup.hs]: Should we `newtype` this? It's only used in this file, so not sure it's strictly necessary.

Comment (alexfmpe) [reply desktop/src/Desktop/Setup.hs]: Heh, it's fine either way. Personally I'd rather not have `Workflow t m` inside the newtype

Comment (mankyKitty) [reply desktop/src/Desktop/Setup.hs]: Sorry, was referring to the `Text` :)

Comment (alexfmpe) [reply desktop/src/Desktop/Setup.hs]: Oh that sounds good, that way we can not have a Show instance.

**Pull request merged**